### PR TITLE
project_timesheet_time_control: Change 'account analytic line' default order

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: project_timesheet_time_control/views/project_task_view.xml [hunk 1]
```diff
@@ -93,6 +93,9 @@
                     </group>
                 </form>
             </xpath>
+            <xpath expr="//field[@name='timesheet_ids']/tree" position="attributes">
+                <attribute name="default_order">date_time</attribute>
```
Comment (pedrobaeza) [inline project_timesheet_time_control/views/project_task_view.xml:97]: Why adding this?

File: project_timesheet_time_control/models/account_analytic_line.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 class AccountAnalyticLine(models.Model):
     _inherit = 'account.analytic.line'
+    _order = 'date_time desc'
```
Comment (pedrobaeza) [inline project_timesheet_time_control/models/account_analytic_line.py]: Only this field? Shouldn't be more?

File: project_timesheet_time_control/views/project_task_view.xml (hunk 1 above)
Comment (ernestotejeda) [reply project_timesheet_time_control/views/project_task_view.xml:97]: The "timesheet_ids" field in the Tasks view form has default_order = "date" specified, so i write that to order timesheets by date_time field insted.

File: project_timesheet_time_control/models/account_analytic_line.py (hunk 2 above)
Comment (ernestotejeda) [reply project_timesheet_time_control/models/account_analytic_line.py]: The only field that I have seen that the account.analytic.line is ordered for is 'date' field field, therefore, I have only set it.

Comment (pedrobaeza) [reply project_timesheet_time_control/models/account_analytic_line.py]: But what about `id`? I think we should add it

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Review by cristinamartinrod — APPROVED** [review]
Tested on Runbot 👍🏻

**Pull request merged**